test/python/transpiler/test_stage_plugin.py

```diff
@@ -51,6 +53,11 @@ def test_list_stage_plugins_invalid_stage_name(self):
         with self.assertRaises(TranspilerError):
             list_stage_plugins("not_a_stage")
 
+    def test_entry_point_obj(self):
+        """Test entry_point_obj function."""
+        basic_obj = entry_point_obj("routing", "basic")
+        self.assertIsInstance(basic_obj, BasicSwapPassManager)
+
```

Comment:
It'd be good to test the failure path as well.

Reply:
Added! Thanks!